dev-util/mingw64-runtime/mingw64-runtime-5.0.1.ebuild

```diff
@@ -78,6 +78,7 @@ src_configure() {
 	CHOST=${CTARGET} econf \
 		--prefix=/usr/${CTARGET} \
 		--includedir=/usr/${CTARGET}/usr/include \
+		$(is_crosscompile && echo --libdir=/usr/${CTARGET}/usr/lib) \
```

Comment:
it is always cross compile, no? so maybe better to have it plain as the includedir?